examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<p>Thanks for contributing! It would also more readable if we break down the functions in smaller components and likewise for other cells.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Sounds good, I'll be making more changes to the notebook, making it more readable and adding more markdown.